monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,8 +16,11 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+[id="cluster-observability-operator-release-notes-0-1-1"]
+== {coo-full} 0.1.1
+This release updates the {coo-full} to support installing the Operator in restricted networks or disconnected environments.
```

Comment:
Is there a special procedure for doing this? In some of my docs there's a special procedure for installing, say, Clair in disconnected environments. If so, you may want to link to that procedure.

Reply:
Good question, @stevsmit. I'll follow up with engineering, but AFAIK, there's nothing. Thanks!

Reply:
@stevsmit - I checked with the eng lead, and he said there's nothing no special procedure or setup required for this specific Operator.